swift/ql/lib/codeql/swift/dataflow/internal/DataFlowPrivate.qll

```diff
@@ -705,7 +709,7 @@ predicate storeStep(Node node1, ContentSet c, Node node2) {
     node1.asExpr() = assign.getSource() and
     node2.(PostUpdateNode).getPreUpdateNode().asExpr() = subscript.getBase() and
     subscript = assign.getDest() and
-    subscript.getBase().getType().(InOutType).getObjectType() instanceof ArrayType and
+//    subscript.getBase().getType().(InOutType).getObjectType() instanceof ArrayType and
```

Comment:
Did you mean to comment this out? Based on https://github.com/github/codeql/pull/13741/commits/42cc6448cc8669879215c3e4ef18624ac50cb97b#r1267210752, I gathered that conjunct was important

Reply:
oh oops